Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -21,9 +47,28 @@ def model(parameters):
     sys.stdout = sys.__stdout__ # Re-enable printing
     print("Simulation completed")
     
-    # Obtain summary statistics from simulation
-    dummy_result = np.array([parameters["gamma"], parameters["reg_param"]]) #Just return parameter values until summary statistics are working
-    return {"data": dummy_result}
+    # Load cellStates
+    pickle_list = helperFunctions.create_pickle_list(export_path)
+    cells = helperFunctions.load_cellStates(export_path, pickle_list[-1]) # load last pickle file
+    
+    # Extract time parameters from simulation (could hard-code to avoid unnecessary repetition)
+    sim_file = os.path.abspath(cellmodeller_module)
+    dt = sim_time_parameters(sim_file)
+    
+    # Calculate summary statistics
+    summary_stats = {}
```

Comment:
The fact that which summary stats are going to be studied should be set by a user. So, summary stats should be defined in the main function by a user so that it's crystal clear which summary stats are going to be studied in this code. For example, if a user wants to add/ remove summary stats from the code, they should search through the code to find where they can do so.

Reply:
I think this is sufficiently transparent